plugins/chat/app/services/chat/auto_join_channels.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
```

Comment:
Even if tested by other means, we at least need one test of the happy path here

Reply:
@jjaffeux / @dbattersby is plugins/chat/spec/services/chat/auto_join_channels_spec.rb what you had in mind?